tests/ports/esp32/ssl_dtls.py

```diff
@@ -0,0 +1,17 @@
+# Very basic test of ssl.SSLContext DTLS.
+
+try:
+    import ssl
```

Comment:
I suggest to use `import tls` because this test will not run on CPython, the `ssl` module doesn't have DTLS support.

Reply:
Done!

Reply:
Actually I don't believe that the ssl or tls module in cpython supports DTLS out of the box in this same way, so I'm a bit confused about how to proceed here for the comparison test.